### PR TITLE
Provide documentation on index statistics

### PR DESCRIPTION
**Pull request by taburet**
Depends on https://github.com/hazelcast/hazelcast/pull/13359

### CODE REVIEW

**Comment by olukas** [comment]
@taburet What do you think about idea to add some sentence, small section or just warning that those stats are for local map and also give users some clue how to count them for whole map? I believe it would be useful. 

Besides that, current docs in this PR looks good to me, thanks for it.

**Comment by taburet** [comment]
@olukas there already exists a small section about that: http://docs.hazelcast.org/docs/latest-dev/manual/html-single/index.html#getting-member-statistics, but I will add the information about your idea with the weighted stats calculation to the index stats section, are you ok with this?

**Comment by olukas** [comment]
@taburet yes, part with weighted stats calculation seems perfect now, thanks! The last objection is missing `customers.addIndex("name", true);` in example as I mentioned above. Once it will be fixed then this PR is ok for me.

File: src/main/java/management/SampleLocalIndexStats.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+import java.util.Map;
+
+public class SampleLocalIndexStats {
+
+    public static void main(String[] args) {
+//tag::lis[]
+        HazelcastInstance hazelcastInstance = Hazelcast.newHazelcastInstance();
+        IMap<String, String> customers = hazelcastInstance.getMap("customers");
+        LocalMapStats mapStatistics = customers.getLocalMapStats();
```
[the remark this answers is not in the document]
Comment (taburet) [reply src/main/java/management/SampleLocalIndexStats.java]: done

**Pull request merged**